packages/devextreme/testing/tests/DevExpress.ui.widgets/chat.tests.js

```diff
@@ -371,6 +370,65 @@ QUnit.module('renderMessage', moduleConfig, () => {
 
         assert.strictEqual($(lastBubble).text(), text);
     });
+
+    QUnit.test('Message list should not run invalidate if 1 new message has been added to items', function(assert) {
```

Comment:
Seems not all scenarios are covered in tests, empty arrays and case when previousValue.length > value.length

Reply:
[cd471fbaf8356ea8ec2656aeb4eebe5e1cfd30e0] and [d012f183f2bdfd6725837ec58bd53e1088d10dea]